### DISCUSSION TITLE
Welcome to Sudoku Discussions!

### DISCUSSION THREAD

**Discussion by Supetorus** [Supetorus/Sudoku#25] [Announcements]
## 👋 Welcome to Sudoku!
Feel free to post commentary and feedback about the current state of the app here. If you find bugs, those would be better to put in issues. When you post a bug, please describe in detail what to do to reproduce the bug.

**Comment by familymamma** [comment]
The green numbers can be hard to see, especially in notes. Green would also make this game unplayable for individuals with blue green color blindness.

**Comment by familymamma** [comment]
Playing in the coral theme - I love the colors, makes me think of summer & watermelon. 

That said, the light colored numbers and writing are heard to see when highlighted and the dark colored- especially the notes - are hard to read when not highlighted. They need more contrast between colors.

**Comment by familymamma** [comment]
Mint theme the dark color is good except when the space is clicked on (disappears when clicked on) but the light color is hard to see - especially the notes.

**Comment by familymamma** [comment]
Colors in Phoniex work great

**Comment by familymamma** [comment]
Dream colors work well. Can’t see what’s in selected square, unless mouse is over it. Otherwise it works well

**Comment by familymamma** [comment]
Neon - I like these colors better then Phoniex (personal preference) can’t see the yellow notes when square is selected- otherwise the colors work great

**Comment by familymamma** [comment]
Mocha, notes hard to see when highlighted. Otherwise very pretty

**Comment by familymamma** [comment]
Dark theme works well too
